index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  
+  <body>
+    <nav id="navbar">
+    <a href="#welcome-sectionn" class="nav-link">section thing</a>
+    <a href="#projectss" class="nav-link">section thing</a>
+    <a href="#About" class="nav-link">section thing</a>
+    </nav>
```

Comment:
Please, try to use identations for each new element, so your code will be more clear for your partners.